packages/core/src/VNodes/AbstractNode.ts

```diff
@@ -16,21 +16,39 @@ export abstract class AbstractNode extends EventMixin {
     readonly id = id;
     _editable: boolean = null;
     /**
-     * True if the node is editable.
+     * True if the node is editable. Propagates to the children.
      * A node that is editable can have its modifiers edited, be moved, removed,
      * and a selection can be made within it.
      * Can be overridden with a `Mode`.
      */
     get editable(): boolean {
-        const attributes = this.modifiers.find(Attributes);
-        const editable = attributes?.get('contenteditable');
-        return typeof this._editable === 'boolean'
-            ? this._editable
-            : [undefined, 'true', ''].includes(editable);
+        if (typeof this._editable === 'boolean') {
+            return this._editable;
+        } else if (this.modifiers.find(Attributes)?.has('contentEditable')) {
+            // TODO: unbind from `Attributes`.
+            return this.contentEditable;
+        } else if (this.parent) {
+            return this.parent.contentEditable || this.parent.editable || false;
+        } else {
+            return true;
+        }
     }
     set editable(state: boolean) {
         this._editable = state;
     }
+    /**
+     * True if the node has the `contentEditable` attribute. This implies that
+     * its children are editable but it is not necessarily itself editable.
+     *
+     * TODO: unbind from `Attributes`.
+     */
+    get contentEditable(): boolean {
+        const editable = this.modifiers.find(Attributes)?.get('contentEditable');
+        return editable === '' || editable?.toLowerCase() === 'true' || false;
+    }
+    set contentEditable(state: boolean) {
+        this.modifiers.get(Attributes).set('contenteditable', '' + state);
+    }
```

Comment:
Attends ça devient ouf là. Déjà y'a `mode.isEditable(node)`, ce qui me semble être le bon truc. Après y'a `node.editable`, que perso je voudrais supprimer parce que si on a le mode bah c'est le mode qui décide si un noeud est éditable, pas le noeud lui-même, autrement franchement qui en dehors de nous va penser à utiliser le mode pour checker qu'un noeud est editable si il voit qu'il existe une propriété qui s'appelle `editable` ? Et alors *en plus* y'a aussi `node.contentEditable` pour rajouter encore une couche de confusion parce que c'est l'attribut "officiel" du DOM qui contrôle ce que nous on a déjà dans les deux autres que j'ai cité avant. Pour moi tout doit disparaitre sauf le mode. Après y'a surement un besoin que j'ai pas compris mais bon, je suis curieux de comprendre... :p

Reply:
I agree that the naming is confusing. But here is the definition of contentEditable by @Zinston 
> contentEditable indique que ses enfants sont éditables, pas lui-même. Par contre il peut triggerer des events.

I also agree with the rest of the comment.